include/config/rcxml.h

```diff
@@ -83,6 +83,7 @@ struct rcxml {
 
 	/* graphics tablet */
 	struct tablet_config {
+		struct tablet_area tablet_area;
```

Comment:
I suggest using `struct wlr_fbox box` here instead. It's more consistent with code-base and avoids having to define another area/box struct.